paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerAdapter.kt

```diff
@@ -159,7 +161,7 @@ interface CustomerAdapter {
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 fun interface CustomerEphemeralKeyProvider {
 
-    suspend fun provide(): Result<CustomerEphemeralKey>
+    suspend fun provide(): CustomerAdapterResult<CustomerEphemeralKey>
```

Comment:
Nit: Can we rename this `provideCustomerEphemeralKey`? Makes things clearer if a developer implements this method with an existing class, e.g. their ViewModel.